spec/queries/get_content_collection_spec.rb

```diff
@@ -429,46 +429,46 @@
 
     context "search in" do
       context "with a single nested field" do
-        let(:search_in) { "details.body" }
+        let(:search_in) { [:'details.body'] }
```

Comment:
Yup, I got taken in by the sheer beauty of symbols and vanity prevailed over legibility. Fixed.